docs/UserGuide.md

```diff
@@ -419,15 +555,15 @@ Optional identifiers can be added to show the list of matched clients with only
 <br><br>
```

Comment:
Also same for the find command examples, we can remove the <br> tags if we do not want to leave lines beween bullet points without images.